src/services/comment.service.ts

```diff
@@ -1,13 +1,22 @@
-import { CommentEntity } from '@/entities';
 import { BaseCommentDto } from '@/dtos';
+import { UpdateCommentRequestDto } from '@/dtos/comments.dto';
+import { CommentEntity } from '@/entities';
 import { HttpException } from '@/exceptions';
+import { updateAndReturn } from '@/utils/queryBuilderUtils';
+
+class CommentService {
+  public async getComments(): Promise<BaseCommentDto[]> {
+    const comments: BaseCommentDto[] = await CommentEntity.find();
+    if (comments.length === 0) throw new HttpException(404, 'No Comments Found');
+    return comments;
+  }
 
-class CommentService{
-    public async getComments(): Promise<BaseCommentDto[]> {
-        const comments: BaseCommentDto[] = await CommentEntity.find()
-        if (comments.length === 0) throw new HttpException(404, 'No Comments Found')
-        return comments
-    }
+  public async updateCommentFromDB(comment_id: number, commentData: UpdateCommentRequestDto): Promise<BaseCommentDto> {
```

Comment:
nit: as I mentioned similar with Matts post, maybe renaming updateComments is concise enough because where else would you update the comment on the server side. Up to you.

Reply:
Yeah me and ray were discussing this earlier. We plan to create a ticket later to make everything follow the same pattern